src/jit/codegencommon.cpp

```diff
@@ -11408,16 +11432,16 @@ unsigned CodeGen::getFirstArgWithStackSlot()
     }
 
     return baseVarNum;
-#elif defined(_TARGET_AMD64_)
+#elif defined(_TARGET_AMDARCH_)
```

Comment:
I don't think this line should change (and there is no `_TARGET_AMDARCH_`)

Reply:
You're right. It's my mistake. Thanks.